PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -21,7 +21,7 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return ((float)numberCorrectAnswers / (float)numberOfQuestions * 100) + "%";
```

Comment:
Looks good! If you want to simplify you can just use a float on the denominator

```suggestion
        return (numberCorrectAnswers / (float)numberOfQuestions * 100) + "%";
```